.nvmrc

```diff
@@ -0,0 +1 @@
+8.9.4
```

Comment:
Do we want to pin so pessimistically? Would it not be better to use `8` as we had in `.travis.yml`?

Reply:
We could do sure, I was just trying to avoid the team having slightly different versions installed and lack of parity with live.